src/main.rs

```diff
@@ -36,13 +36,13 @@ fn main() -> PassResult<()> {
 
     let pm = PassManager::default_passes()?;
 
-    // list all the avaliable pass options when pass-help subcommand is used
-    if let Some(sub) = opts.sub {
+    // handle pass-help and format
+    if let Some(ref sub) = opts.sub {
```

Comment:
I don't see a good reason to use `ref` here because we don't need to keep `opts.sub` as a reference for later use. But if this is desired, better to match on `&opts.sub` instead since that is clearer than using the less common `ref`